[2022-03-14 00:19:33,621] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: 2_daily_transformation_analysis.daily_analysis manual__2022-03-14T00:19:19.188463+00:00 [queued]>
[2022-03-14 00:19:33,632] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: 2_daily_transformation_analysis.daily_analysis manual__2022-03-14T00:19:19.188463+00:00 [queued]>
[2022-03-14 00:19:33,633] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-14 00:19:33,634] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-03-14 00:19:33,634] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-14 00:19:33,647] {taskinstance.py:1264} INFO - Executing <Task(BashOperator): daily_analysis> on 2022-03-14 00:19:19.188463+00:00
[2022-03-14 00:19:33,654] {standard_task_runner.py:52} INFO - Started process 900 to run task
[2022-03-14 00:19:33,657] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', '2_daily_transformation_analysis', 'daily_analysis', 'manual__2022-03-14T00:19:19.188463+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/transform_and_analysis.py', '--cfg-path', '/tmp/tmpsp1uywgm', '--error-file', '/tmp/tmp1gvgfvzv']
[2022-03-14 00:19:33,659] {standard_task_runner.py:77} INFO - Job 9: Subtask daily_analysis
[2022-03-14 00:19:33,711] {logging_mixin.py:109} INFO - Running <TaskInstance: 2_daily_transformation_analysis.daily_analysis manual__2022-03-14T00:19:19.188463+00:00 [running]> on host 1910aafd392e
[2022-03-14 00:19:33,798] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=2_daily_transformation_analysis
AIRFLOW_CTX_TASK_ID=daily_analysis
AIRFLOW_CTX_EXECUTION_DATE=2022-03-14T00:19:19.188463+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-03-14T00:19:19.188463+00:00
[2022-03-14 00:19:33,800] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-03-14 00:19:33,801] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'cd /dbt && dbt run --models analysis --profiles-dir .']
[2022-03-14 00:19:33,811] {subprocess.py:85} INFO - Output:
[2022-03-14 00:19:36,271] {subprocess.py:89} INFO - Running with dbt=0.20.0
[2022-03-14 00:19:39,095] {subprocess.py:89} INFO - Found 5 models, 0 tests, 0 snapshots, 0 analyses, 338 macros, 0 operations, 3 seed files, 0 sources, 0 exposures
[2022-03-14 00:19:39,097] {subprocess.py:89} INFO - 
[2022-03-14 00:19:40,936] {subprocess.py:89} INFO - 00:19:40 | Concurrency: 200 threads (target='dev')
[2022-03-14 00:19:40,938] {subprocess.py:89} INFO - 00:19:40 |
[2022-03-14 00:19:40,989] {subprocess.py:89} INFO - 00:19:40 | 1 of 2 START view model analysis.hotel_count_by_day.................. [RUN]
[2022-03-14 00:19:40,991] {subprocess.py:89} INFO - 00:19:40 | 2 of 2 START view model analysis.thirty_day_avg_cost................. [RUN]
[2022-03-14 00:19:42,418] {subprocess.py:89} INFO - 00:19:42 | 1 of 2 OK created view model analysis.hotel_count_by_day............. [[32mSUCCESS 1[0m in 1.43s]
[2022-03-14 00:19:42,425] {subprocess.py:89} INFO - 00:19:42 | 2 of 2 OK created view model analysis.thirty_day_avg_cost............ [[32mSUCCESS 1[0m in 1.43s]
[2022-03-14 00:19:43,110] {subprocess.py:89} INFO - 00:19:43 |
[2022-03-14 00:19:43,111] {subprocess.py:89} INFO - 00:19:43 | Finished running 2 view models in 4.01s.
[2022-03-14 00:19:43,147] {subprocess.py:89} INFO - 
[2022-03-14 00:19:43,149] {subprocess.py:89} INFO - [32mCompleted successfully[0m
[2022-03-14 00:19:43,151] {subprocess.py:89} INFO - 
[2022-03-14 00:19:43,152] {subprocess.py:89} INFO - Done. PASS=2 WARN=0 ERROR=0 SKIP=0 TOTAL=2
[2022-03-14 00:19:44,413] {subprocess.py:93} INFO - Command exited with return code 0
[2022-03-14 00:19:44,439] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=2_daily_transformation_analysis, task_id=daily_analysis, execution_date=20220314T001919, start_date=20220314T001933, end_date=20220314T001944
[2022-03-14 00:19:44,476] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-14 00:19:44,503] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
